src/models/StuntSheetDot.ts

```diff
@@ -27,4 +27,16 @@ export default class StuntSheetDot extends Serializable<StuntSheetDot> {
     super();
     this.fromJson(dotJson);
   }
+
+  xAtBeat(beat: number): number {
+    return this.cachedFlow !== null && beat < this.cachedFlow.length
+      ? this.cachedFlow[beat].x
+      : this.x;
+  }
+
+  yAtBeat(beat: number): number {
+    return this.cachedFlow !== null && beat < this.cachedFlow.length
+      ? this.cachedFlow[beat].y
+      : this.y;
+  }
```

Comment:
Sorry that I hadn't written this out, but when I was thinking this out here's what I was thinking:

At beat 0 (the "Hup!" count), this should return the dot's x/y position.

At beats 1 (left leg strikes the ground) - end of stuntsheet, this should return `cachedFlow[beat - 1]`

If `cachedFlow.length` is less than the number of beats for the stuntsheet, then return the last position on `cachedFlow` when `beat > cachedFlow.length`

If `cachedFlow` does not exist, then it should be generated and stored.

I can implement this in a future PR since this was what I was planning on doing anyways :laughing: So you can either address it yourself or I can do it

Reply:
So does that mean that there is only a beat 0 for stunt sheet 1?

Reply:
Lets address with:
Set the shows to have beats be [0, N) #100